chat-components/src/main/java/com/pubnub/components/chat/service/channel/OccupancyService.kt

```diff
@@ -2,13 +2,14 @@ package com.pubnub.components.chat.service.channel
 
 import androidx.compose.runtime.compositionLocalOf
 import com.pubnub.components.chat.ui.component.presence.Presence
+import com.pubnub.framework.data.ChannelGroupId
 import com.pubnub.framework.data.ChannelId
 import com.pubnub.framework.data.Occupancy
 import com.pubnub.framework.data.UserId
 import kotlinx.coroutines.flow.Flow
 
 interface OccupancyService {
-    fun bind()
+    fun bind(channels: List<ChannelId> = emptyList(), channelGroups: List<ChannelGroupId> = emptyList())
```

Comment:
@kazydek `OccupancyService.bind` method takes two lists now

Reply:
There's no such detailed info on it in the docs.